index.html

```diff
@@ -284,5 +284,36 @@ <h4>Iswan Dracho</h4>
             <button class="btn-large-orange">View All</button>
         </div>
     </section>
+    <section id="testimonials-section">
+        <div class="testimonials-section__container">
+            <h2>What Customers Say</h2>
+            <div class="testimonials-section__testimonials">
+                <div class="testimonials-section__testimonial">
+                    <img src="./assets/customer-2.png" alt="Starla Virgoun">
+                    <h4>Starla Virgoun</h4>
+                    <h5>Financial Advisor</h5>
+                    <p>
+                        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta repudiandae cupiditate eaque obcaecati! 
+                    </p>
+                </div>
+                <div class="testimonials-section__testimonial">
+                    <img src="./assets/customer-1.png" alt="James Steven">
+                    <h4>James Steven</h4>
+                    <h5>Uber Driver</h5>
+                    <p>
+                        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta repudiandae cupiditate eaque obcaecati! 
+                    </p>
+                </div>
+                <div class="testimonials-section__testimonial">
+                    <img src="./assets/customer-3.png" alt="Tosh Mat">
+                    <h4>Tosh Mat</h4>
```

Comment:
Wrong teacher name: change to Tosh Mattew

Reply:
Good catch, I'll fix it.

Reply:
lgtm